e2e/protractor/suites/actions/copy-move/destination-picker-dialog.test.ts

```diff
@@ -1,353 +0,0 @@
-    searchFolderId = (await userApi.nodes.createFolder(searchFolder, destinationId)).entry.id;
-    searchSubFolder1Id = (await userApi.nodes.createFolder(searchSubFolder1, searchFolderId)).entry.id;
-    await userApi.nodes.createFolder(searchSubFolder2, searchSubFolder1Id);
-
-    await userApi.sites.createSite(site, SITE_VISIBILITY.PRIVATE);
-    const docLibId = await userApi.sites.getDocLibId(site);
-    searchFolderSiteId = (await userApi.nodes.createFolder(searchFolder, docLibId)).entry.id;
-    searchSubFolder1SiteId = (await userApi.nodes.createFolder(searchSubFolder1, searchFolderSiteId)).entry.id;
-    await userApi.nodes.createFolder(searchSubFolder2, searchSubFolder1SiteId);
-
-    await userApi.sites.addSiteConsumer(site, consumer);
-    await userApi.sites.addSiteContributor(site, contributor);
-    await userApi.sites.addSiteCollaborator(site, collaborator);
-
-    fileIdConsumer = (await consumerApi.nodes.createFile(file)).entry.id;
-    fileIdContributor = (await contributorApi.nodes.createFile(file)).entry.id;
-    fileIdCollaborator = (await collaboratorApi.nodes.createFile(file)).entry.id;
-
-    await adminApiActions.login();
-    adminFolderId = (await adminApiActions.nodes.createFolder(adminFolder)).entry.id;
-
-    await userApi.search.waitForNodes(searchFolder, { expect: 2 });
-  });
-
-  afterAll(async () => {
-    await userActions.login(username, username);
-    await userActions.deleteNodes([fileId, destinationId]);
-    await userActions.deleteSites([site]);
-
-    await consumerApi.nodes.deleteNodeById(fileIdConsumer);
-    await contributorApi.nodes.deleteNodeById(fileIdContributor);
-    await collaboratorApi.nodes.deleteNodeById(fileIdCollaborator);
-
-    await adminApiActions.login();
-    await adminApiActions.deleteNodes([adminFolderId]);
-  });
-
-  afterEach(async () => {
-    await page.closeOpenDialogs();
-    await page.dataTable.clearSelection();
-  });
-
-  describe('general', () => {
-    beforeAll(async () => {
-      await loginPage.loginWith(username);
-    });
-
-    beforeEach(async () => {
-      await page.dataTable.selectItem(file);
-      await page.toolbar.clickMoreActionsCopy();
-      await dialog.waitForDialogToOpen();
-    });
-
-    it('[C263875] Dialog UI', async () => {
-      expect(await dialog.getDialogTitle()).toEqual(`Copy '${file}' to...`);
-      expect(await dialog.searchInput.isPresent()).toBe(true, 'Search input is not displayed');
-      expect(await dialog.isSelectLocationDropdownDisplayed()).toBe(true, 'Select Location dropdown not displayed');
-      expect(await breadcrumb.currentFolder.getText()).toEqual('Personal Files');
-      expect(await dataTable.isItemPresent(destination)).toBe(true, 'Personal Files content not displayed');
-      expect(await dialog.isCopyButtonEnabled()).toBe(true, 'Copy button is not disabled');
-      expect(await dialog.isCancelButtonEnabled()).toBe(true, 'Cancel button is not enabled');
```

Comment:
isCancelButtonEnabled function can be removed probably too as it seems unused

Reply:
Removed isCancelButtonEnabled()